v3/src/components/graph/adornments/lsrl/lsrl-adornment-model.ts

```diff
@@ -75,6 +104,18 @@ export const LSRLAdornmentModel = AdornmentModel
       }
     })
     return caseValues
+  },
+  get lineDescriptions() {
+    const lineDescriptions: ILineDescription[] = []
+    self.lines.forEach((linesArray, key) => {
+      linesArray.forEach(line => {
+        const { category, intercept, slope } = line
+        if (!intercept || !isFinite(intercept) || !slope || !isFinite(slope)) return
```

Comment:
```suggestion
        if (!Number.isFinite(intercept) || !Number.isFinite(slope)) return
```
`intercept` and `slope` can be zero for valid lines.